### PR TITLE
Move Vector Tiles module from Community to Extension (GSIP-138)

### PR DESCRIPTION
**Pull request by DBlasby**
Followed the documentation (http://docs.geoserver.org/stable/en/developer/policies/community-modules.html#promoting-a-community-module)
for moving a Community module to an Extension.

### CODE REVIEW

File: src/extension/vectortiles/pom.xml [hunk 1]
```diff
@@ -25,7 +25,6 @@
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.3.1</version>
```
Comment (tbarsballe) [inline src/extension/vectortiles/pom.xml]: ~~Why is version removed here?~~

Nevermind, I see it is moved up to `src/pom`

**Comment by DBlasby** [comment]
Corrected the (c) headers

**Comment by tbarsballe** [comment]
Looks good. Just waiting on Travis CI tests to run now.

**Comment by aaime** [comment]
Looks good here too, Travis is happy, merging, keep an eye on the nightly build to see if the packaging works properly

**Pull request merged**